### PR TITLE
add(worldview) resolution debugger overlay and test resolutions

### PR DESCRIPTION
**Pull request by chrisgervang**
Adding a `debug` prop to `Map` component when debugging resolutions. Related to #159

Screen Shot 2021-10-05 at 10 58 26 AM

### CODE REVIEW

File: examples/worldview/src/features/map/DebugOverlay.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+
+/* eslint-disable complexity */
+export function DebugOverlay({containerRef, deckRef, dimension, previewSize, dpi, onDpiChange}) {
+  const container = containerRef && containerRef.current;
```
Comment (hkim8567) [inline examples/worldview/src/features/map/DebugOverlay.js:5]: would it be better to use optional chaining? ``` const container = containerRef?.current;```
same with other variables till line 9

File: examples/worldview/src/scenes/sftrees/index.js [hunk 2]
```diff
@@ -35,7 +35,7 @@ export const useScene = () => {
     // layerKeyframes: [],
     // tripKeyframe: {},
     cameraKeyframe: {
-      timings: [500, 3500],
+      timings: [500, 2500],
```
Comment (hkim8567) [inline examples/worldview/src/scenes/sftrees/index.js:38]: what's the reason for this change?

File: examples/worldview/src/scenes/sftrees/index.js [hunk 3]
```diff
@@ -49,8 +49,15 @@ export const useScene = () => {
...
     );
-    dispatch(resolutionChange('1920x1080'));
+    // dispatch(resolutionChange('1920x1080'));
     // dispatch(resolutionChange({width: 3840, height: 2160}));
+    // dispatch(resolutionChange({width: 5760, height: 5760}));
+    // dispatch(resolutionChange({width: 7680, height: 4320}));
+    // dispatch(resolutionChange({width: 1920, height: 1920}));
+    // dispatch(resolutionChange({width: 1080, height: 1920}));
+    // dispatch(resolutionChange({width: 1280, height: 720}));
+    dispatch(resolutionChange({width: 320, height: 180}));
```
Comment (hkim8567) [inline examples/worldview/src/scenes/sftrees/index.js:59]: same here

File: examples/worldview/src/scenes/sftrees/index.js (hunk 2 above)
Comment (chrisgervang) [reply examples/worldview/src/scenes/sftrees/index.js:38]: Don't need to wait an extra second anymore when testing the animation.

File: examples/worldview/src/features/map/DebugOverlay.js (hunk 1 above)
Comment (chrisgervang) [reply examples/worldview/src/features/map/DebugOverlay.js:5]: Yeah, I just haven't adopted it yet since last I checked it was "draft" syntax. Maybe it's official now?
